lib/elixir/pages/getting-started/sigils.md

```diff
@@ -235,6 +235,6 @@ iex> ~i(42)n
 -42
 ```
 
-Custom sigils may be either a single lowercase character or several uppercase characters.
+Custom sigils may be either a single lowercase character or a uppercase character followed by more uppercase characters and digits.
```

Comment:
```suggestion
Custom sigils may be either a single lowercase character, or an uppercase character followed by more uppercase characters and digits.
```

Without the oxford comma it's a bit ambiguous if its:

- a single lowercase character
- an uppercase character followed by more uppercase characters and/or digits

or
- a single lowercase character or uppercase character - followed by more uppercase characters and digits.


Alternatively, a bulleted list would remove all ambiguity.

Reply:
I agree, the bulleted list makes sense to me. I will commit your suggest at the first very least as a well placed oxford comma can indeed make all the difference :)